src/lib/format/l20n/ast/parser.js

```diff
@@ -485,7 +487,7 @@ export default {
 
     const junk = new AST.JunkEntry(
       this._source.slice(this._curEntryStart, nextEntry));
-    if (this._config.pos) {
+    if (AST.Node._config.pos) {
```

Comment:
I agree. In the updated version I moved it back and actually turned `setPosition` to be a method on the parser itself that turns into a no-op if `_config.pos` is false.